### PR TITLE
Floating button Button replaced

### PR DESCRIPTION
**Pull request by marizvi**
Hello sir,
I have successfully replaced the floating button with material button to display the event details,
Since I am still at the learning stage and I know that this change is very basic but still in coming days I will definitely try my best to come up with more changes for this app to make it look more attractive.
If everything is working fine then I will suggest you to please merge my small piece of code.

### CODE REVIEW

File: lib/ui/homePage(temporary).dart [hunk 1]
```diff
@@ -78,24 +82,51 @@ class _MyHomePageState extends State<MyHomePage> {
...
+          //     onPressed: (){
+          //        _incrementCounter;
+          // Navigator.of(context).pushNamed(EventDetailsPage.routeName);
+          //     }, 
+          //   child: Text("Event Details"),
+            
+          //   style: ButtonStyle(
+          //     backgroundColor: MaterialStateProperty.all(Colors.purple[500]),
+          //   ),
+          //   )
```
Comment (vikiwarrior) [inline lib/ui/homePage(temporary).dart:98]: Remove this commented code

Comment (marizvi) [reply lib/ui/homePage(temporary).dart:98]: oh sorry...

Comment (marizvi) [reply lib/ui/homePage(temporary).dart]: Removed...

**Comment by vikiwarrior** [comment]
Remove unused variables and make similar buttons for the Event Page, Feedback Page, and past event Page.

**Comment by marizvi** [comment]
Ok, I will do that.

**Comment by marizvi** [comment]
Sir but these 3 pages are not yet been created, so I will be leaving onpressed: (){} as a blank function for those buttons.
Is that fine?

**Comment by vikiwarrior** [comment]
> Sir but these 3 pages are not yet been created, so I will be leaving onpressed: (){} as a blank function for those buttons. Is that fine?

yep

**Comment by marizvi** [comment]
As you told, I removed unused variables like (increment counter etc.), also added three more buttons on the temporary homescreen.
If anything else is missing here then kindly let me know and,
If everything is fine then please merge my part of code as well, so that I can look forward to make this app more attractive in near future by generating more issues.

Thanks..

**Review by vikiwarrior — APPROVED** [review]

**Comment by marizvi** [comment]
Sir,
What else is needed in order to get this PR merged?

**Pull request merged**